sandbox/controller.go

```diff
@@ -22,6 +22,9 @@ import (
 	"github.com/containerd/containerd/api/services/sandbox/v1"
 )
 
+const DefaultSandboxController = "podsandbox"
```

Comment:
If the mode is type, I  think we should use const type and add validation for this.

```go
type Mode string

const (
     ModeDefault Mode = "default"
     ModeRemote = "remote"
)

func ValidateMode() error {
    ...
}
```

Something like that.

Reply:
I want make "SandboxMode" similar to "Snapshotter",  they're all string type. If we change it to Mode type, there wolud be some conversion. Is it necessary?